README.md

```diff
@@ -1,18 +1,38 @@
-# ProjectBabble
-A Mouth tracking system for VR, 100% Opensource!
 
-Currently, the architecture is a modified efficientnetv2-b0 model (although we actively look into other architectures and models) training on blendshapes from ARKit (with some of our own modifcations to the ARKit standard)!  The model outputs 45 blenshapes of which most cross over with SRanipals blendshapes.
+![Babble Logo](https://github.com/SummerSigh/ProjectBabble/blob/SummerSigh-patch-4/Babble.png?raw=true)
 
-### How to run
-Go to the releases tab, download the latest exe. Run the exe installer and enjoy!
+# Table of Contents
+- [What is Babble?](#what-is-babble)
+- [Features](#features)
+- [How do I set it up?](#setup-more-detailed-wiki-and-setup-video-coming-soon)
+- [Useful Links](#useful-links)
+  
 
-### Warranty
-Lol 
+## What is Babble?
 
-### Credit 
+<p align="center">
+Babble is an opensource mouth tracking project designed to work with any existing VR headset. We strive to make our models robust to different lighting, cameras, image qualities, and facial structures!
+</p>
 
-Thank you to dfgHiatus#7426 for providing the unity demo as well as the intergrations with other apps!
+## Features
+- 100% Opensource! 🌟
+- Fast and robust! 🚀
+- Works with most existing blendshape standards! ⚙️
+- Constantly updated and modified! 🔧
 
-Thank you to Rames The Generic#3540 who made the dataset, tested and evaluated the performence for the models, and for being awesome!
+## Setup (More detailed wiki and setup video coming soon!) 
+To install babble is fairly simple! Head over to the releases tab and download the EXE located there. Run the EXE and install babble! After that you can test by doing the following steps: 
```

Comment:
Should probably clarify that these instructions are for testing with a desktop web camera.

Reply:
ok done